legend-engine-xt-analytics-search-generation/src/main/java/org/finos/legend/engine/generation/SearchDocumentArtifactGenerationExtension.java

```diff
@@ -56,8 +56,7 @@ public boolean canGenerate(PackageableElement element)
     @Override
     public List<Artifact> generate(PackageableElement element, PureModel pureModel, PureModelContextData data, String clientVersion)
     {
-        String path = getElementFullPath(element, pureModel.getExecutionSupport());
-        String fileName = String.format("%s.json", path);
+        String fileName = "SearchDocumentResult.json";
```

Comment:
Should we define this as a constant? (That can then be referenced in the tests)

Reply:
Updated